src/olympia/amo/sitemap.py

```diff
@@ -92,6 +97,45 @@ def location(self, item):
         return reverse(item)
 
 
+class CategoriesSitemap(Sitemap):
+    priority = 0.7
+    # i18n = True  # TODO: support all localized urls
```

Comment:
I know it's a TODO but some localized URLs are excluded, see: https://github.com/mozilla/addons-frontend/issues/6644. It'd be interesting to know exactly which URLs should be part of the sitemap because I think it should be all URLs for which we have canonical URLs (which is a subset of _all possible URLs_)

Reply:
#16909 for localized urls, but I'd not really worked out _which_ urls.  (I didn't know we'd already had that discussion in 2018!)